src/components/InteractionLayer/index.js

```diff
@@ -177,29 +177,32 @@ class InteractionLayer extends React.Component {
   }
 
   onMouseDown = e => {
-    e.preventDefault();
-    this.mouseDown = true;
-    const xpos = e.nativeEvent.offsetX;
-    const ypos = e.nativeEvent.offsetY;
-    this.setState({
-      area: {
-        id: Date.now(),
-        start: this.getDataForCoordinate(xpos, ypos, true),
-      },
-    });
+    if (this.props.onAreaDefined) {
+      this.mouseDown = true;
+      const xpos = e.nativeEvent.offsetX;
+      const ypos = e.nativeEvent.offsetY;
+      this.setState({
+        area: {
+          id: Date.now(),
+          start: this.getDataForCoordinate(xpos, ypos, true),
+        },
+      });
+    }
   };
 
-  onMouseUp = e => {
-    setTimeout(() => {
-      this.mouseUp = false;
-      this.dragging = false;
-    }, 50);
-    e.preventDefault();
-    const { area } = this.state;
-    if (area.start && area.end && isLargeEnough(area)) {
-      this.props.onAreaDefined(area);
+  onMouseUp = () => {
+    const { onAreaDefined } = this.props;
+    if (onAreaDefined) {
```

Comment:
Edge case;
Can we potentially get stuck here? If the application toggles area-creation off mid-drag for some reason you won't be able to get rid of the area. Could we set `area` to null whether `onAreaDefined` is defined or not?